basket/news/utils.py

```diff
@@ -280,9 +280,10 @@ def language_code_is_valid(code):
 ]
 
 
-def get_user_data(token=None, email=None, extra_fields=None, get_fxa=False):
+def get_user_data(token=None, email=None, payee_id=None, extra_fields=None, get_fxa=False):
```

Comment:
This should be fine but generally speaking I'm careful adding more parameters out of order, especially in a function with all optional parameters. I did a quick search and the only place where we don't explicitly name the parameter is in https://github.com/mozmeao/basket/blob/79624a385bcef27c639c312916282b02d11edac6/basket/news/tasks.py#L386, so maybe you can change that to `get_user_data(token=data.get('token'), email=data.get('email'), extra_fields=['id']))` just to be on the safe side.

Reply:
Ah, nice catch!

I did think maybe `get_user_data` needs a refactor - maybe taking the (now) three identifiers and expecting a dict instead - something like `{ 'type': 'token', 'value': '123456' }`. Anyway, I'll leave that for later clean up.